### PR TITLE
Tmsagarofficial patch 1

### PR DESCRIPTION
**Pull request by tmsagarofficial**
## Description
Grammar and typo fixes
## References
This is just a simple PR to address the issues as mentioned in the descrption

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [NA] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [NA] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: stores/abac-with-rebac/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ type document
         define can_view : viewer from published
 ```
 
-If the attribute is a discrete variable with many possible values, e.g. birth date, age, IP address, a a currency amount, then it's not possible to be modeled with pure ReBAC and you need to resort to using [OpenFGA conditions](https://openfga.dev/docs/modeling/conditions). 
+If the attribute is a discrete variable with many possible values, e.g. birth date, age, IP address, and currency amount, then it's not possible to be modeled with pure ReBAC and you need to resort to using [OpenFGA conditions](https://openfga.dev/docs/modeling/conditions).
```
Comment (aaguiarz) [inline stores/abac-with-rebac/README.md:54]:
```suggestion
If the attribute is a discrete variable with many possible values, e.g. birth date, age, IP address, a currency amount, then it's not possible to be modeled with pure ReBAC and you need to resort to using [OpenFGA conditions](https://openfga.dev/docs/modeling/conditions). 
```

**Review by aaguiarz — APPROVED** [review]
Thanks @tmsagarofficial!!

It looks great!

**Comment by tmsagarofficial** [comment]
Thanks @aaguiarz

**Comment by tmsagarofficial** [comment]
Can hacktoberfest label be added to this? @aaguiarz

**Comment by aaguiarz** [comment]
@tmsagarofficial just added it

**Comment by tmsagarofficial** [comment]
@aaguiarz Ohh okay Thanks , I see that this repo doesn't have hacktoberfest label 😅, In this case adding `hacktoberfest-accepted` would help me make this contribution count towards some streak.

**Comment by aaguiarz** [comment]
@tmsagarofficial I just added the hacktoberfest / hacktoberfest-accepted  labels to the repo and to the PR.

Does this work?

**Comment by tmsagarofficial** [comment]
Yeah thanks mate

**Pull request merged**